cmd/api/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/textileio/go-tableland/pkg/database"
 	"github.com/textileio/go-tableland/pkg/eventprocessor"
 	"github.com/textileio/go-tableland/pkg/eventprocessor/eventfeed"
-	"github.com/textileio/go-tableland/pkg/readstatementresolver"
```

Comment:
I removed the `readstatementresolver` package. It was created to solve a cyclical dependency issue. Now the `readstatementresolver` is part of the `parsing` package.